packages/core/README.md

```diff
@@ -1,6 +1,20 @@
 # @uma/core
 
-This package contains contract artifacts for all UMA smart contracts.
+This package contains contract artifacts for all UMA smart contracts. It contains the following sets of smart contracts within the `contracts` directory:
+
+| **Directory Name**            | **Description**                                                                                                                                                                                                                    |
+| ----------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
check out this sweet table I added to explain the directories within the core contracts folder.

Reply:
nice 👍